src/components/datepicker/DateRangePicker.js

```diff
@@ -171,10 +169,10 @@ class DatePicker extends React.Component {
     }
 
     updateStateTo(to, toText) {
```

Comment:
updateStateTo and updateStateFrom is principal the same only setState differs .... which could be given as funciton for common method updateState.